frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -343,16 +348,13 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     setDetailsItem(item);
     setShowDetails(true);
 
-    if (ignoreWarning) {
-      localStorage.setItem(COMMUNITY_PROVIDERS_WARNING_LOCAL_STORAGE_KEY, 'true');
+    if (loaded && ignoreWarning) {
+      setIgnoreOperatorWarning(true);
     }
   };
 
   const openOverlay = (item: OperatorHubItem) => {
-    const ignoreWarning =
-      localStorage.getItem(COMMUNITY_PROVIDERS_WARNING_LOCAL_STORAGE_KEY) === 'true';
-
-    if (!ignoreWarning && item.providerType === COMMUNITY_PROVIDER_TYPE) {
+    if (!ignoreOperatorWarning && item.providerType === COMMUNITY_PROVIDER_TYPE) {
```

Comment:
If the API is slow, would this not cause a modal to appear? I'll see if I can reproduce this...

Reply:
Ah the rest of the page has plenty of time to load before you click on the tile that _may_ cause this... so I guess it's extremely unlikely we'll have that and it's not a blocking modal so I think this is fine.

Reply:
I can reproduce this if you refresh and quickly click the tile.

Here's a PR I created to fix this: https://github.com/rottencandy/console/pull/1

Reply:
Thanks, merged.

Reply:
After speaking with andrew, we agreed that the having the modal is the better approach in case it's taking so long to load.